### PR TITLE
News article for after SR2020 Finals

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Fixes https://github.com/srobo/tasks/issues/412
Fixes https://github.com/srobo/tasks/issues/413

Note: we should also put #228 live at the same time.

### CODE REVIEW

File: _posts/2020-07-25-post-competition.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+three weekends, with matches being live-streamed from within a simulator.
+While this changed many aspects of the competition the core challenge to create
+an autonomous robot remained the same.
+
+The Final
+---------
+
+This year's final featured four teams who had all proven their capabilities
+throughout the competition. All robots got off to a strong start, though
+JAMDynamics and Lawrence Sheriff School soon took a lead. However later in the
```
Comment (antoinepetty) [inline _posts/2020-07-25-post-competition.md:36]:
```suggestion
JAMDynamics and Lawrence Sheriff School soon took a lead. However, later in the
```

File: _posts/2020-07-25-post-competition.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+match, the "poisoning" effect of having mixed tokens in a scoring zone changed
+everything leaving all four teams with the same number of points.
+
+<!-- Video of Final -->
+<iframe
+  class="center video"
+  src="https://www.youtube-nocookie.com/embed/XXX-TODO"
```
Comment (antoinepetty) [inline _posts/2020-07-25-post-competition.md:43]: May need to wait up to a day before the stream finishes processing and can be downloaded for cutting

File: _posts/2020-07-25-post-competition.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  frameborder="0"
+  width="100%"
+  allowfullscreen
+  >
+</iframe>
+
+This tie for first place lead to a tie-breaker match, played between the same
+four teams though with their starting corners changed. Once again the
+reliability of these four robots showed, with each collecting a token inside the
+one minute mark. In the end it was JAMDynamics robot which was able to collect
```
Comment (antoinepetty) [inline _posts/2020-07-25-post-competition.md:53]:
```suggestion
one minute mark. In the end it was JAMDynamics' robot which was able to collect
```
Not sure whether it's this or `JAMDynamics's`

File: _posts/2020-07-25-post-competition.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+three silver tokens, including one stolen from Hills Road Sixth Form College
+second teams' scoring zone, which took victory.
```
Comment (antoinepetty) [inline _posts/2020-07-25-post-competition.md:55]:
```suggestion
second team's scoring zone, which took victory.
```

File: _posts/2020-07-25-post-competition.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+Having these tokens within their scoring zone (i.e. their corner of the arena)
+at the end of the match would earn them 3 points apiece. However, if a robot
+gathered more than one token colour within their scoring zone, each token's
+value drops from 3 points to 1 point. Full details of the game and the awards
+available this year are available in the [rulebook][rulebook].
+
+Our competition event this year was rather different to normal as COVID-19
+prevented our usual weekend event. Instead the competition took place over
+three weekends, with matches being live-streamed from within a simulator.
+While this changed many aspects of the competition the core challenge to create
```
[the remark this answers is not in the document]
Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md]: 6c4a4c3

File: _posts/2020-07-25-post-competition.md (hunk 1 above)
Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md]: 0446bf7

File: _posts/2020-07-25-post-competition.md (hunk 3 above)
Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md]: db7041e

File: _posts/2020-07-25-post-competition.md (hunk 4 above)
Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md]: d75917f

File: _posts/2020-07-25-post-competition.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+<iframe
+  class="center video"
+  src="https://www.youtube-nocookie.com/embed/XXX-TODO"
+  frameborder="0"
+  width="100%"
+  allowfullscreen
+  >
+</iframe>
+
+While we weren't able to see them in person, we saw some brilliant robot designs
```
Comment (RealOrangeOne) [inline _posts/2020-07-25-post-competition.md:67]: They looked the same. I'd be tempted to either drop the "design" or mention that it was more about strategy design etc

```suggestion
While we weren't able to see them in person, we saw some brilliant robots
```

File: _posts/2020-07-25-post-competition.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | [SWI: South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)
+| <span class="has-explanation" title="The first rookie team1that demonstrates a moving robot to the community will be awardedwith an edible prize at the final competition">First Robot Movement</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA))
+| <span class="has-explanation" title="The team that is judged to have the best online presence using the hashtag #srobo19">Online Presence</span> |  [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)
+
+With a dominating 111 league points, Kenilworth Sixth Form not only placed
+highest among the rookie teams in the league but placed highest overall.
```
Comment (RealOrangeOne) [inline _posts/2020-07-25-post-competition.md:89]: This reads almost like they took 1st.

```suggestion
highest among the rookie teams but placed highest overall in the league.
```

File: _posts/2020-07-25-post-competition.md (hunk 6 above)
Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md:67]: Ah, I was trying to acknowledge the physical robots here.

File: _posts/2020-07-25-post-competition.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+competition and after lockdown began as well as for their activy on the forums.
+
+For full details on all the awards, please see the [rulebook][rulebook].
+
+You can see a breakdown of scores for each match, as well as the overall league
+ranking on the [competition website](/comp/).
+
+Of course, this competition would not be possible without the tireless work of
+our [volunteers][volunteers], who developed the [simulated world][simulator] the
+competition was run in and enabled us to switch to a virtual competiton.
```
Comment (RealOrangeOne) [inline _posts/2020-07-25-post-competition.md:113]: I'd be interested in a CTA on this

File: _posts/2020-07-25-post-competition.md (hunk 6 above)
Comment (RealOrangeOne) [reply _posts/2020-07-25-post-competition.md:67]: ah, yeah definitely worth drawing focus to that, but worth being slightly more explicit

File: _posts/2020-07-25-post-competition.md (hunk 8 above)
Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md]: 5a6a594

File: _posts/2020-07-25-post-competition.md (hunk 6 above)
Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md:67]: 98bf644

File: _posts/2020-07-25-post-competition.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+This year's final featured four teams who had all proven their capabilities
+throughout the competition. All robots got off to a strong start, though
+JAMDynamics and Lawrence Sheriff School soon took a lead. However, later in the
+match, the "poisoning" effect of having mixed tokens in a scoring zone changed
+everything leaving all four teams with the same number of points.
+
+<!-- Video of Final -->
+<iframe
+  class="center video"
+  src="https://www.youtube-nocookie.com/embed/XXX-TODO"
```
Comment (antoinepetty) [inline _posts/2020-07-25-post-competition.md:43]:
```suggestion
  src="https://www.youtube-nocookie.com/embed/MAXpgODIqiI"
```

Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md]: Thanks! Already did this (and removed the other embed) in fcdf8d7.

File: _posts/2020-07-25-post-competition.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)
+| <span class="has-explanation" title="The first rookie team1that demonstrates a moving robot to the community will be awardedwith an edible prize at the final competition">First Robot Movement</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA))
+| <span class="has-explanation" title="The team that is judged to have the best online presence using the hashtag #srobo19">Online Presence</span> |  [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)
+
+With a dominating 111 league points, Kenilworth Sixth Form not only placed
+highest among the rookie teams but placed highest overall in the league.
+Especially given the changes to the competition through the year, beating other
+more eperience teams is a remarkable feat.
```
Comment (sedders123) [inline _posts/2020-07-25-post-competition.md:83]:
```suggestion
more experience teams is a remarkable feat.
```

File: _posts/2020-07-25-post-competition.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)
+| <span class="has-explanation" title="The first rookie team1that demonstrates a moving robot to the community will be awardedwith an edible prize at the final competition">First Robot Movement</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA))
+| <span class="has-explanation" title="The team that is judged to have the best online presence using the hashtag #srobo19">Online Presence</span> |  [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)
+
+With a dominating 111 league points, Kenilworth Sixth Form not only placed
+highest among the rookie teams but placed highest overall in the league.
+Especially given the changes to the competition through the year, beating other
+more experience teams is a remarkable feat.
```
Comment (sedders123) [inline _posts/2020-07-25-post-competition.md:83]: Apologies my typo fix still was a typo
```suggestion
more experienced teams is a remarkable feat.
```

File: _posts/2020-07-25-post-competition.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+-------
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | JAM: JAMDynamics
+| 2nd Place             | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)
+| 3rd Place             | [LSS: Lawrence Sheriff School](http://www.lawrencesheriffschool.net/)
+| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | [SWI: South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/)
+| <span class="has-explanation" title="The first rookie team1that demonstrates a moving robot to the community will be awardedwith an edible prize at the final competition">First Robot Movement</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA))
```
Comment (RealOrangeOne) [inline _posts/2020-07-25-post-competition.md:77]:
```suggestion
| <span class="has-explanation" title="The first rookie team that demonstrates a moving robot to the community">First Robot Movement</span> | [KSF: Kenilworth Sixth Form](https://www.ksn.org.uk/) ([watch their robot](https://www.youtube.com/watch?v=KuFwqVyn_YA))
```

File: _posts/2020-07-25-post-competition.md [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+| <span class="has-explanation" title="The team that is judged to have the best online presence using the hashtag #srobo19">Online Presence</span> |  [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)
```
Comment (RealOrangeOne) [inline _posts/2020-07-25-post-competition.md:78]:
```suggestion
| <span class="has-explanation" title="The team that is judged to have the best online presence using the hashtag #srobo2020">Online Presence</span> |  [MAI: Gymnasium Markt Indersdorf](https://mai-robotics.de/)
```

File: _posts/2020-07-25-post-competition.md (hunk 11 above)
Comment (RealOrangeOne) [inline _posts/2020-07-25-post-competition.md:83]:
```suggestion
more experienced teams is a remarkable feat.
```

Comment (PeterJCLaw) [reply _posts/2020-07-25-post-competition.md]: e78f0cf

**Pull request merged**